tst/EnergyPlus/unit/UnitaryHybridAirConditioner.unit.cc

```diff
@@ -402,19 +402,16 @@ TEST_F(EnergyPlusFixture, Test_UnitaryHybridAirConditioner_Unittest)
     EXPECT_FALSE(ErrorsFound);                    // expect no errors
     DataZoneEquipment::GetZoneEquipmentData(*state);    // read zone equipment    SystemReports::ReportMaxVentilationLoads();
     DataZoneEquipment::ZoneEquipInputsFilled = true;
-    ZoneOAMassFlow.allocate(state->dataGlobal->NumOfZones);
-    ZoneOAMass.allocate(state->dataGlobal->NumOfZones);
-    ZoneOAVolFlowStdRho.allocate(state->dataGlobal->NumOfZones);
-    ZoneOAVolFlowCrntRho.allocate(state->dataGlobal->NumOfZones);
-    ZoneOAVolStdRho.allocate(state->dataGlobal->NumOfZones);
-    ZoneOAVolCrntRho.allocate(state->dataGlobal->NumOfZones);
-    ZoneMechACH.allocate(state->dataGlobal->NumOfZones);
+    state->dataSysRpts->ZoneOAMassFlow.allocate(state->dataGlobal->NumOfZones);
+    state->dataSysRpts->ZoneOAMass.allocate(state->dataGlobal->NumOfZones);
+    state->dataSysRpts->ZoneOAVolFlowStdRho.allocate(state->dataGlobal->NumOfZones);
+    state->dataSysRpts->ZoneOAVolFlowCrntRho.allocate(state->dataGlobal->NumOfZones);
+    state->dataSysRpts->ZoneOAVolStdRho.allocate(state->dataGlobal->NumOfZones);
+    state->dataSysRpts->ZoneOAVolCrntRho.allocate(state->dataGlobal->NumOfZones);
+    state->dataSysRpts->ZoneMechACH.allocate(state->dataGlobal->NumOfZones);
     MAT.allocate(state->dataGlobal->NumOfZones);
     ZoneAirHumRatAvg.allocate(state->dataGlobal->NumOfZones);
-    MaxHeatingLoadMetByVent.allocate(state->dataGlobal->NumOfZones);
-    MaxOverheatingByVent.allocate(state->dataGlobal->NumOfZones);
-    MaxCoolingLoadMetByVent.allocate(state->dataGlobal->NumOfZones);

```

Comment:
@mitchute These lines got lost in the shuffle here. Replacing them should fix the unit test failure.

Reply:
Thanks for the 🦅  👁️  @mjwitte. I'm confused why this didn't fail on my local machine 😕  

I've pushed a fix here: https://github.com/NREL/EnergyPlus/pull/8432/commits/890fd55d26356a38537c1669e1b3aaece84bde6c

Reply:
@mitchute Is it possible that debug builds on Mac and Linux don't catch array bounds violations? The system report stuff in this unit test where it failed on array bounds just happens to fire, but the results aren't important for the test.

Reply:
I'd have to check when/where the array bounds violations are checked. I just ran it before and after the fix it in release on Mac and it sailed through without complaining both times.

Reply:
"release" - that explains it. Try a debug build and it should fail.

Reply:
No bounds checking in release

Reply:
Got it. Confirmed that it does fail in debug locally. Thanks, @mjwitte!